app/assets/stylesheets/application.sass.scss

```diff
@@ -89,39 +89,52 @@ $color_app-pale-blue: #ccdff1;
   }
 }
 
-.app-header__navigation-text {
-  @include nhsuk-font(16);
-  border-bottom: nhsuk-spacing(1) solid transparent;
-  border-top: nhsuk-spacing(1) solid transparent;
-  color: $color_nhsuk-blue;
-  display: block;
-  padding: 12px nhsuk-spacing(3);
-  text-decoration: none;
-
-  @include mq($from: large-desktop) {
-    color: $color_nhsuk-white;
-    line-height: normal;
-  }
+.app-logo {
+  margin-right: 12px;
 
-  .nhsuk-icon__chevron-right {
-    fill: $color_nhsuk-grey-3;
-    position: absolute;
-    right: nhsuk-spacing(1);
-    top: 11px;
+  @include mq($from: tablet) {
+    margin-right: 0;
   }
 }
 
-.app-header__menu-toggle--no-search {
-  @extend .nhsuk-header__menu-toggle;
-  @include mq($until: desktop) {
-    right: 0px;
+.app-header {
+  &__container {
+    border-bottom: 1px solid rgba(255, 255, 255, 0.2);
+  }
+
+  &__service-name {
+    display: inline-block;
+    line-height: 1.25;
+    vertical-align: top;
+
+    @include mq($from: tablet, $until: large-desktop) {
+      max-width: 300px;
+    }
+  }
+
+  &__navigation-list {
+    justify-content: flex-end;
+
+    .button_to,
+    .app-button-link {
+      height: 100%;
+    }
+  }
+
+  &__navigation-text {
+    @extend .nhsuk-header__navigation-link;
+    @include nhsuk-font(16);
+    text-decoration: none;
   }
 }
 
 .app-button-link {
   @extend .nhsuk-header__navigation-link;
-  background: none;
+  @include nhsuk-font(16);
```

Comment:
Did `font-size: inherit` not work? Wondering how much CSS this `@include` adds where it may not be needed?

Reply:
Nope; with `font-size: inherit`:

![image](https://github.com/nhsuk/record-childrens-vaccinations/assets/1650875/cd34e1e5-2218-44f1-a8c6-6d7e9e5833ee)